AAI/AAIConnectProfile.md

```diff
@@ -338,12 +338,60 @@ skinparam componentStyle rectangle
 left to right direction
 
 package "Unspecified clients, additional services, protocols" {
-component "<b>Visa Assertion Source</b> (1)\norganisation" as VisaSource1
-component "<b>Visa Assertion Source</b> (2)\norganisation" as VisaSource2
-component "<b>Visa Assertion Source</b> (...)\norganisation" as VisaSourceN
+component "<b>Visa Assertion Source</b> (1)\norganization" as VisaSource11
+component "<b>Visa Assertion Source</b> (2)\norganization" as VisaSource12
+component "<b>Visa Assertion Source</b> (1)\norganization" as VisaSource21
+component "<b>Visa Assertion Source</b> (2)\norganization" as VisaSource22
 
-database "<b>Visa Assertion</b>\n<b>Repository</b>\nabstract service" as VisaRepository
-component "<b>Visa Issuer</b>\nabstract service\n(optional)" as VisaIssuer
+database "<b>Visa Assertion</b>\n<b>Repository (1)</b>\nabstract service" as VisaRepository1
+component "<b>Visa Issuer (1)</b>\nabstract service\n(optional)" as VisaIssuer1
+
+database "<b>Visa Assertion</b>\n<b>Repository (2)</b>\nabstract service" as VisaRepository2
+component "<b>Visa Issuer (2)</b>\nabstract service\n(optional)" as VisaIssuer2
+}
+
+package "Specified GA4GH AAI clients, services, protocols" {
+component "<b>Broker</b>\nservice" as Broker #FAFAD2
+component "<b>Client</b>\napplication" as Client #FAFAD2
+component "<b>Passport</b>\n<b>Clearinghouse</b>\nservice" as ClearingHouse #FAFAD2
+}
+
+VisaSource11 ..> VisaRepository1
+VisaSource12 ..> VisaRepository1
+VisaRepository1 ..> VisaIssuer1
+
+VisaSource21 ..> VisaRepository2
+VisaSource22 ..> VisaRepository2
+VisaRepository2 ..> VisaIssuer2
+
+
+VisaRepository1 ..> Broker 
+VisaIssuer1 ..> Broker 
+VisaRepository2 ..> Broker 
+VisaIssuer2 ..> Broker 
+Broker --> Client 
+Client --> ClearingHouse
+
+@enduml
+# OR, MORE SEPARATED:
```

Comment:
I prefer the original non-separated diagram.

Reply:
We could go back to a non-separated approach, but consider the new diagrams.